docs/release/k3s-release.md

```diff
@@ -0,0 +1,253 @@
+# K3S Kubernetes Patch Release Process
+This document details the K3S kubernetes patch release process.
+For reference, this document makes the following assumptions:
```

Comment:
Does the doc need to make these assumptions? I think we can write it in such a way without the assumptions and provide examples where necessary.

Reply:
It doesn't have to make these assumption. However I put them in to provide context to the release. The origin document from which this is based greatly lacked context of this nature. Any thoughts on how to restructure to make this more implicit?

Reply:
Remove the assumptions for now and that should be fine.

Reply:
This is done.